### ISSUE TITLE
Improve debug message when command returns 0

### ISSUE THREAD

**Issue by MarkMaldaba** [apache/cordova-cli#412]
# Feature Request

## Motivation Behind Feature

The debug output* from the command-line build process (e.g. with the `--verbose` option) outputs the exit code of commands that are run.  When the command fails you get output such as the following, which is as expected:

````
Command finished with error code 1: <details>
````

However, when the command indicates success with an exit code of zero, the following output is generated:

````
Command finished with error code 0: <details>
````

This is confusing, as this is not an error code, but a success code.  It makes scanning for errors harder and implies that there has been a problem with the build.

(* note this may also affect normal output - I haven't checked.)

## Feature Description

Expected output, for result code of zero only:
````
Command finished with success code 0: <details>
````

If it's problematic to vary the output depending on whether it returns a zero or not, an alternative wording for use in all cases would be:

````
Command finished with exit code 1: <details>
````

Either way, a change to the text to stop it implying that a success code is an error code would be very helpful.

**Comment by raphinesse** [comment]
Good point. The line is [logged here](https://github.com/apache/cordova-common/blob/469e7b4a49a1a39c7fc22aa206730b9d58613467/src/superspawn.js#L127). I would appreciate a PR that changes the wording from _error code_ to _exit code_
